docs/usage/custom-resources.md

```diff
@@ -96,6 +96,52 @@ with the
 [`core.labelSources`](../reference/worker-configuration-reference.md#corelabelsources)
 configuration option.
 
+## NodeFeatureGroup
+
+> **NOTE:** This feature is experimental and may change in the future.
+
+NodeFeatureGroup is an NFD-specific custom resource that is designed for
+grouping Nodes by matching features rules. NFD-Master watches for
+NodeFeatureGroupRules objects in the cluster and creates a NodeFeatureGroup
+object for each NodeFeatureGroupRule. The NodeFeatureGroup object contains a
+list of nodes that match the NodeFeatureGroupRule.
+
+```yaml
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureGroup
+metadata:
+  name: my-sample-group-resource
+spec:
+  FeatureGroup:
+    name: my-sample-group-resource
+    nodes:
+    - node-1
+    - node-2
+    - node-3
+```
+
+NodeFeatureGroupRule is an NFD-specific custom resource that is designed for
+grouping Nodes by matching features rules. NFD-Master watches for
+NodeFeatureGroupRules objects in the cluster and creates a NodeFeatureGroup
+object for each NodeFeatureRule.
```

Comment:
This is not correct wrt. to the latest design. There is only one type of CRD object (NFG)